src/Dofs/ConstraintHandler.jl

```diff
@@ -33,14 +33,14 @@ which applies the condition via [`apply!`](@ref) and/or [`apply_zero!`](@ref).
 """
 struct Dirichlet # <: Constraint
     f::Function # f(x) or f(x,t) -> value(s)
-    facets::Union{Set{Int},Set{FacetIndex},Set{FaceIndex},Set{EdgeIndex},Set{VertexIndex}}
+    facets::Union{OrderedSet{Int},OrderedSet{FacetIndex},OrderedSet{FaceIndex},OrderedSet{EdgeIndex},OrderedSet{VertexIndex}}
```

Comment:
```suggestion
    facets::OrderedSet{T} where {T<:Union{Int, FacetIndex, FaceIndex, EdgeIndex, VertexIndex}}
```
To make a bit easier to read? 
(Not sure if this works on all julia versions though...) 

Alternatively `::Union{OrderedSet{Int}, OrderedSet{<:BoundaryIndex}}` with a slight (but probably negligible performance overhead)